### PR TITLE
Refactor: test component

### PR DESCRIPTION
**Pull request by MatthijsSmets**

### CODE REVIEW

File: src/app/test/test.component.ts [hunk 1]
```diff
@@ -65,106 +66,94 @@ export class TestComponent implements OnInit, AfterViewInit {
...
-  }
-
-  getCopiedReports(): void {
-    this.httpService.getTestReports(this.currentView.metadataNames, this.currentView.storageName).subscribe({
-      next: (response: TestListItem[]) => this.addCopiedReports(response),
-      error: () => catchError(this.errorHandler.handleError()),
-    });
-  }
-
   loadData(path: string): void {
```
Comment (MarkvdVorst) [inline src/app/test/test.component.ts:97]:
```suggestion
  loadData(path?: string): void {
```
Wouldn't it be better to have this as an optional paramater? Currently I only see one place where this method is called where it was not an empty string.

File: src/app/test/test.component.html [hunk 2]
```diff
@@ -3,25 +3,37 @@
...
+      </button>
+      <button data-cy-test="copySelected" class="btn btn-info" title="Copy Selected"
+              (click)="copySelected()"><i class="fa fa-copy"></i></button>
+      <button data-cy-test="downloadBinary" class="btn btn-info" title="Download Binary"
+              (click)="downloadSelected()"><i class="fa fa-download"></i></button>
+      <button data-cy-test="upload" type="button" class="btn btn-info" title="Upload"
+              (click)="uploadFileTest.click()">
         <i class="fa fa-upload"></i>
         <input #uploadFileTest data-cy-test="uploadFile" type="file" id="uploadFileTest"
                (change)="uploadReport($event)">
```
Comment (MarkvdVorst) [inline src/app/test/test.component.html:39]: Is it necessary to have this extra input element? Wouldn't it be better to just the button or just the input element or am I missing something?

File: src/app/test/test.component.ts (hunk 1 above)
Comment (MarkvdVorst) [reply src/app/test/test.component.ts:97]: If we do this we can remove most of the falsy empty strings from the other method calls giving cleaner, more readable code.

**Review by MarkvdVorst — CHANGES_REQUESTED** [review]
Looks good. I noted down some small points which may need some improvements if it's possible.

File: src/app/test/test.component.html (hunk 2 above)
Comment (MatthijsSmets) [reply src/app/test/test.component.html:39]: Yeah the input element is there to allow native browser file uploads

Comment (MarkvdVorst) [reply src/app/test/test.component.html:39]: Alright. Good to know.

File: src/app/test/test.component.ts (hunk 1 above)
Comment (MatthijsSmets) [reply src/app/test/test.component.ts:97]: Oh right, good one, changed it.

**Review by MarkvdVorst — APPROVED** [review]

**Pull request merged**